packages/examples/src/config.ts

```diff
@@ -74,16 +65,14 @@ registerExamples([
     label: 'Configuration (Default)',
     data,
     schema,
-    uiSchema: uischema,
-    translations
+    uiSchema: uischema
```

Comment:
While we are it, could we unify casing here, so using `uischema` instead of `uiSchema`?

Reply:
sure